linkerd/proxy/api-resolve/src/pb.rs

```diff
@@ -56,6 +60,19 @@ fn to_id(pb: TlsIdentity) -> Option<identity::Name> {
     }
 }
 
+pub(in crate) fn to_authority(o: AuthorityOverride) -> Option<Authority> {
+    match o.authority_override.parse() {
+        Ok(name) => Some(name),
+        Err(_) => {
+            tracing::warn!(
+                "Ignoring invalid authority override: {}",
+                o.authority_override
+            );
```

Comment:
Do we `warn` elsewhere in this file? Generally, if there's nothing a user can do about it, I prefer to keep this at debug. IMO, if the destination service is providing invalid values, that's an internal error and not a user error.